web/pgadmin/browser/static/js/browser.js

```diff
@@ -366,6 +372,74 @@ define('pgadmin.browser', [
       scripts[n].push({'name': m, 'path': p, loaded: false});
     },
     masterpass_callback_queue: [],
+    getMenuList: function(name, item, d, skipDisabled=false) {
+      let obj = this;
+      let category = {
+        'common': []
+      };
+      for(let _key of Object.keys(obj.menus[name][d._type])){
+        if(obj.menus[name][d._type][_key].category in this.menu_categories) {
```

Comment:
Create a var for `obj.menus[name][d._type][_key].category` and use it to simplify code.